src/components/GeneralSettingsSection.vue

```diff
@@ -26,188 +26,226 @@
-			</h3>
-
-			<label class="backup-settings__time-slots">
-				{{ t('backup', 'Create restoring points during the following time slot during the day:') }}
-				<select v-model.number="settings.timeSlotsStart"
-					:disabled="loadingFetchSettings"
-					name="timeSlotsStart"
-					@change="setSettings">
-					<option v-for="(hour, index) in new Array(24)" :key="index" :value="index | leadingZero">{{ index | leadingZero }}:00</option>
-				</select>
-				{{ t('backup', 'and') }}
-				<select v-model.number="settings.timeSlotsEnd"
+			<div class="backup-settings__cron-enabled">
+				<h3 class="backup-settings__sub-headers">
+					{{ t('backup', 'Enable background tasks') }}
+					<span v-if="loadingSetSettings > 0" class="icon-loading" />
+				</h3>
+
+				<div class="backup-settings__info">
+					{{ t('backup', 'You can enable background task for backups. This means that the creation, maintenance and purges of backups will be done automatically.') }}
+					<ul>
+						<li>
+							{{ t('backup', 'Creation: new restoring points will be created according the the schedule.') }}
+						</li>
+
+						<li>
+							{{ t('backup', 'Maintenance: restoring points will be packed and copied to potential external storages.') }}
+						</li>
+						<li>
+							{{ t('backup', 'Purge: old restoring points will be deleted automatically according the the retention policy.') }}
+						</li>
+					</ul>
+				</div>
+
+				<CheckboxRadioSwitch
+					:loading="loadingFetchSettings"
 					:disabled="loadingFetchSettings"
-					name="timeSlotsEnd"
-					@change="setSettings">
-					<option v-for="(hour, index) in new Array(24)" :key="index" :value="index | leadingZero">{{ index | leadingZero }}:00</option>
-				</select>
-			</label>
-
-			<CheckboxRadioSwitch
-				:loading="loadingFetchSettings"
-				:disabled="loadingFetchSettings"
-				:checked.sync="settings.allowWeekdays"
-				@update:checked="setSettings">
-				{{ t('backup', 'Allow to create full restoring points during weekday') }}
-			</CheckboxRadioSwitch>
-
-			<ul class="backup-settings__delays">
-				<li>
-					{{ t('backup', 'Delay between two full restoring points') }}:
-					<input v-model.number="settings.delayFullRestoringPoint"
-						:disabled="loadingFetchSettings"
-						class="backup-settings__input"
-						type="text"
-						@change="setSettings"> {{ n('backup', 'day', 'days', settings.delayFullRestoringPoint) }}
-				</li>
-
-				<li>
-					{{ t('backup', 'Delay between two partial restoring points') }}:
-					<input v-model.number="settings.delayPartialRestoringPoint"
-						:disabled="loadingFetchSettings"
-						class="backup-settings__input"
-						type="text"
-						@change="setSettings"> {{ n('backup', 'day', 'days', settings.delayPartialRestoringPoint) }}
-				</li>
-			</ul>
-		</form>
+					:checked.sync="settings.cronEnabled"
+					@update:checked="setSettings">
+					{{ t('backup', 'Enable background tasks for restoring point creation, maintenance and purge.') }}
```

Comment:
maybe `Enable background tasks to automatically manage creation, maintenance and purge`